packages/manager/core/generator/app/conditional-templates/onboarding/onboarding.test.ts.hbs

```diff
@@ -0,0 +1,16 @@
+import { expect, test } from '@playwright/test';
+import { login, logout } from '../../../../../playwright/login';
+import * as config from '../../../../../playwright/config.json';
+
+test('Onboarding test', async ({ page }) => {
+  await page.goto(config.appUrl);
+  await login(page);
+  const onboardingPath = '/#/onboarding';
+  const absoluteUrl = new URL(onboardingPath, config.appUrl).href;
+  await page.click(':nth-child(1) > a');
+  await page.goto(absoluteUrl, { waitUntil: 'networkidle' });
```

Comment:
what is the purpose of a click then a goto? if you forgot an assertion then split this in two tests, if not this test should be improved

Reply:
Is it because there is a problem if we don't do the click and then the goto, caused by the login redirection
Maybe I can call you on monday about it to show you ?

Reply:
Done ✅ Fixed in login.ts